app/models/subscription.rb

```diff
@@ -0,0 +1,10 @@
+class Subscription < ApplicationRecord
+  belongs_to :contact
+  belongs_to :list
+
+  enum status: %i[pending_create created pending_destroy]
+
+  after_initialize do
+    self.status ||= :pending_create if new_record?
+  end
```

Comment:
There's code that specifically sets `pending_create`, can we let that do this -- we can leave status as `nil` or we could have some default status that is `not_set`.  As is above, if you create a new Subscription object for test purposes it will be `pending_create` which is not necessarily true.